lib/helpers/helpers-html.js

```diff
@@ -157,6 +157,16 @@ module.exports.register = function (Handlebars, opts) {
       }
       figure += '</figure>';
       return new Handlebars.SafeString(figure);
+    },
+
+    /**
+     * {{#striptags}}{{{data}}}{{{//striptags}}}
+     * Remove all HTML tags from the nested context
+     * @param  {[type]} options [description]
+     * @return {[type]}         Text with HTML tags removed
+     */
+    striptags: function (options) {
```

Comment:
`options` are what handlebars passes as the last argument automatically. If you're planning on passing in a `context` when using the helper in a template, then you want to do `striptags: function (context, options) {`.

Your test below says `{{#striptags context options}}` but it's not actually testing that.